patterns/page-home-business.php

```diff
@@ -14,5 +14,13 @@
 <!-- wp:pattern {"slug":"twentytwentyfour/text-feature-grid-3-col"} /-->
 <!-- wp:pattern {"slug":"twentytwentyfour/text-alternating-images"} /-->
 <!-- wp:pattern {"slug":"twentytwentyfour/testimonial-centered"} /-->
-<!-- wp:pattern {"slug":"twentytwentyfour/posts-grid-2-col"} /-->
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:heading {"align":"wide","style":{"typography":{"lineHeight":"1"},"spacing":{"margin":{"top":"0"}}},"fontSize":"x-large"} -->
+	<h2 class="wp-block-heading alignwide has-x-large-font-size" style="margin-top:0;line-height:1"><?php esc_html_e( 'Watch, Read, Listen', 'twentytwentyfour' ); ?></h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:pattern {"slug":"twentytwentyfour/posts-3-col"} /-->
```

Comment:
If this is used as part of `front-page.html` while the WordPress site is configured to display a "static front page", it would look odd, since in that case there would only be the one page, with its excerpt shown in one of the three columns.

I'm not sure how we could fix this without modifying the WordPress template hierarchy. Honestly, I find this an odd configuration of WordPress core to choose `front-page` regardless of whether that page is configured to show the blog or a static page.

Potentially we can use the `frontpage_template_hierarchy` filter to customize this for TT4 to prefer a custom template, e.g. something like:
```php
add_filter(
	'frontpage_template_hierarchy',
	function ( $templates ) {
		if ( is_home() ) {
			array_unshift( $templates, 'front-page-home.php' );
		}
		return $templates;
	}
);
```

If we did that in `functions.php`, we could have both a `front-page.html` and `front-page-home.html` template for the individual situations. `front-page-home.html` could look use the content as is here, while `front-page.html` would instead have content somewhat similar to `page.html`, i.e. displaying the content of a single individual page.

Reply:
I would like to see a video of how that would work with the Site Editor. Why is it a PHP file? How do users edit it?

Reply:
I'm getting more confused the more we talk about it. I assumed this was an issue so I renamed it, and now it seems to be a bigger issue that it's renamed? I'm fine reverting this change, but I have a hard time understanding what would be the best case, as it feels like whatever we do it's not. Adding code to fix something is not my preferred idea, because then you have to further explain how things work, and it will be harder for the average user to work with TT4. Since it's a legacy product, that's not what I want to achieve.

Reply:
I think this shouldn't be fixed in the theme

Reply:
> we should probably limit this PR to only fixing the home.html template (without renaming it to front-page.html) and then continue exploring a solution for the renaming separately.


I'm good with this as well; to keep the scope/discussion focused.

Reply:
I agree with you all that fixing it in the theme would be merely a workaround for it being fixed in core, so I'm totally supportive of not doing that. We just won't be able to use this template as `front-page.html` until that part _is_ fixed in core. I'm happy to open a Trac ticket for that part, so that at least in the future it could be reconsidered.

@luminuu 
> I'm getting more confused the more we talk about it. I assumed this was an issue so I renamed it, and now it seems to be a bigger issue that it's renamed?

It's an issue that the blog template only shows the blog posts buried far down on the page, but that's a separate and _minor_ issue compared to what I flagged in #705 / https://core.trac.wordpress.org/ticket/59759.

Given the above limitations of core, I think we should for now stick to only fixing the `home.html` template, _without_ renaming it. In other words, that _won't_ address #404, but it _will_ address #705 / https://core.trac.wordpress.org/ticket/59759.